umh-core/README.md

```diff
@@ -1,178 +1,219 @@
-                │   ┌───────────────────────┐   │
-                │   │       Agent         │   │
-                │   │ - Reads bootstrapped  │   │
-                │   │   YAML config         │   │
-                │   │ - Manages state via   │   │
-                │   │   a deterministic FSM │   │
-                │   │ - Updates desired state│   │
-                │   │   on YAML changes     │   │
-                │   └───────────────────────┘   │
-                │           │                   │
-                │   ┌───────────────────────┐   │
-                │   │   Benthos Pipelines   │   │
-                │   │ - Protocol converters │   │
-                │   │ - Data bridges        │   │
-                │   └───────────────────────┘   │
-                │           │                   │
-                │   ┌───────────────────────┐   │
-                │   │      Redpanda         │   │
-                │   │  (Store-and-Forward)  │   │
-                │   └───────────────────────┘   │
-                │           │                   │
-                │  (Status & Topic Data sent   │
-                │   to UMH Management Console) │
-                └───────────────────────────────┘
+/data
+ ├─ config.yaml           # Desired-state YAML (watched continuously)
+ ├─ logs/                 # Rolling logs for agent, every DFC, Redpanda …
+ ├─ redpanda/             # Broker data & WALs (backup-worthy)
+ └─ hwid                  # Device fingerprint sent to the console
 ```
 
-## Getting Started
-
-### 1. Prepare the Bootstrap Configuration
+Mount **one persistent volume** (e.g. `umh-core-data`) to `/data` and you’re done.
 
-On first startup, **umh-core** uses a minimal bootstrapped configuration YAML file. This file is intended to be updated later—either externally via a CI/CD pipeline or by the agent’s communication process—so that the desired state of the container (and its pipelines) can change dynamically.
+---
 
-Create a file named `umh-lite-bootstrap.yaml`:
+## 4  Configuration File (what *you* edit)
 
 ```yaml
-# umh-lite-bootstrap.yaml
+agent:
+  metricsPort: 8080
+  communicator:
+    apiUrl: https://management.umh.app/api
+  releaseChannel: stable
+  location:
+    0: My-Enterprise
+    1: My-Plant
+    2: My-Line
+
+dataFlow:
+  - name: hello-world-dfc
+    desiredState: active
+    dataFlowComponentConfig:
+      benthos:
+        input:
+          generate:
+            mapping: root = "hello world from DFC!"
+            interval: 1s
+        pipeline:
+          processors:
+            - bloblang: root = content()
+        output:
+          stdout: {}
+```
 
-apiKey: "YOUR-UMH-BACKEND-API-KEY"
+#### 4.1  What’s **not** in the file?
 
-metadata:
-  deviceId: "edge-device-001"
-  location: "enterprise.siteA.lineB"
+Everything under the top-level key `internal:` is reserved for UMH engineers (built-in services, monitors, etc.).
+You never touch it; in the console UI those fields stay hidden.
```

Comment:
"You should never touch it", "You shall never touch it", "You may never touch it", ...